app/controllers/katello/api/v2/systems_bulk_actions_controller.rb

```diff
@@ -182,8 +182,6 @@ def available_incremental_updates
 
     def find_errata
       @errata = Katello::Erratum.where(:uuid => params[:errata_ids])
-      not_found = params[:errata_ids] - @errata.pluck(:uuid)
-      fail _("Could not find all specified errata ids: %s") % not_found.join(', ') unless not_found.empty?
```

Comment:
the issue you hit was handling when param[:errata_ids] was nil.  But you could do that just by:

params[:errata_ids] ||= []

or something similar